src/valkey-benchmark.c

```diff
@@ -835,7 +855,15 @@ static void showLatencyReport(void) {
         printf("  %d bytes payload\n", config.datasize);
         printf("  keep alive: %d\n", config.keepalive);
         if (config.cluster_mode) {
-            printf("  cluster mode: yes (%d primaries)\n", config.cluster_node_count);
+            const char *node_prefix = NULL;
+            if (config.read_from_replica == FROM_ALL) {
+                node_prefix = "all";
```

Comment:
The 'all' name here does not reflect the same semntics as was done before. for example when I use all I will get something like:
```
Cluster has 6 all nodes:

all 0: fcc9d096ad7dfa9f365c917941201a5bf037eb20 127.0.0.1:7000
all 1: 7cd62de6bb802699dad0c33961b2542c88afbeca 127.0.0.1:7004
all 2: 7330c7461582a447520745518f2d3a53ea2a5b0c 127.0.0.1:7001
all 3: f8b7e3188b0ec5bc6ea1a52342733dbb3dde2680 127.0.0.1:7005
all 4: 1a70c15fad7fc590092bce033dbc09b0cc057a86 127.0.0.1:7002
all 5: 4b1228f49237178f92583d72ec1c6d06e452eaed 127.0.0.1:7003
```

which does not sound right. we should probably reflect the correct node role taken after parsing the cluster nodes.